### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by cduhn17**
#  #

## 🗣 Description ##

Update default CODEOWNERS

## 💭 Motivation and context ##

We have more CODEOWNERS than what is listed inthe CODEOWNERS

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: backend/src/api/app.ts [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+    }
+    // Only allow global admins to access all other paths.
+    const user = (await auth.authorize({
+      authorizationToken: req.cookies['crossfeed-token']
+    })) as auth.UserToken;
+    if (user.userType !== UserType.GLOBAL_ADMIN) {
+      return res.status(401).send('Unauthorized');
+    }
+    return next();
+  },
```
[the remark this answers is not in the document]
Comment (Matthew-Grayson) [reply backend/src/api/app.ts:252]: Resolved

[the remark this answers is not in the document]
Comment (Matthew-Grayson) [reply backend/src/api/app.ts:252]: Resolved

File: backend/src/api/app.ts [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+      authorizationToken: req.headers.authorization
+    })) as auth.UserToken;
+    if (
+      user.userType !== UserType.GLOBAL_VIEW &&
+      user.userType !== UserType.GLOBAL_ADMIN
+    ) {
+      return res.status(401).send('Unauthorized');
+    }
+    return next();
+  },
```
[the remark this answers is not in the document]
Comment (Matthew-Grayson) [reply backend/src/api/app.ts:270]: Resolved

File: backend/src/api/app.ts [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+  peProxy
+);
+
+// Routes that require an authenticated user, without
+// needing to sign the terms of service yet
+const authenticatedNoTermsRoute = express.Router();
+authenticatedNoTermsRoute.use(checkUserLoggedIn);
```
[the remark this answers is not in the document]
Comment (Matthew-Grayson) [reply backend/src/api/app.ts:277]: Resolved

File: backend/src/api/app.ts [hunk 4]
```diff
@@ -0,0 +1,465 @@
...
+);
+authenticatedNoTermsRoute.put('/users/:userId', handlerToExpress(users.update));
+
+app.use(authenticatedNoTermsRoute);
+
+// Routes that require an authenticated user that has
+// signed the terms of service
+const authenticatedRoute = express.Router();
+
+authenticatedRoute.use(checkUserLoggedIn);
```
[the remark this answers is not in the document]
Comment (Matthew-Grayson) [reply backend/src/api/app.ts:292]: Resolved

**Comment by cduhn17** [comment]
@dav3r , @jsf9k 
Disregard request to review on this PR.

**Review by Matthew-Grayson — APPROVED** [review]
Looks Great! All CodeQL issues are resolved and the infrastructure pipeline looks good to go on the staging side.

**Review by aloftus23 — APPROVED** [review]

**Review by rapidray12 — APPROVED** [review]
Allows us to turn the crank of the LZ Post Deployment team forking/pulling this repo

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -3,7 +3,9 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+
+
+* @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj @rapidray12 @schmelz21 @cduhn17 @aloftus23 @Matthew-Grayson @nickviola
```
Comment (dav3r) [inline .github/CODEOWNERS]: Remove owners who are not involved in this repo and alphabetize those who are:
```suggestion
* @aloftus23 @cduhn17 @Matthew-Grayson @nickviola @rapidray12 @schmelz21
```

**Review by dav3r — CHANGES_REQUESTED** [review]
Please take care of one small, but important change before I approve.

**Review by dav3r — APPROVED** [review]
Approved without full review or addressing all linter issues at the request of @cduhn17 so that downstream work can proceed.

Please address the remaining issues as soon as possible in subsequent PRs.

**Pull request merged**